rest_framework_serializer_extensions/utils.py

```diff
@@ -71,3 +73,31 @@ def internal_id_from_model_and_external_id(model, external_id):
         raise model.DoesNotExist
 
     return instance_id
+
+
+def model_from_definition(model_definition):
+    """
+    Return a Django model corresponding to model_definition.
+
+    Model definition can either be a string defining how to import the model,
+    or a model class.
+
+    Arguments:
+        model_definition: (str|django.db.models.Model)
+
+    Returns:
+        (django.db.models.Model)
+    """
+    if isinstance(model_definition, six.string_types):
+        model = import_local(model_definition)
+    else:
+        model = model_definition
+
+    try:
+        assert issubclass(model, models.Model)
+    except (AssertionError, TypeError):
+        raise AssertionError(
+            '"{0}"" is not a Django model'.format(model_definition)
+        )
+
+    return model
```

Comment:
Looks good to me. I think a few tests cases showing how this work and how extensions deals with deserialization in general would be useful in highlighting any use cases where this would be problematic